labels/en-US.json

```diff
@@ -238,6 +238,21 @@
   "deleteQuoteName": "Delete Quote",
   "deliverTo": "Deliver To",
   "deliverYourItem": "Thanks for letting us know you're here! Please fill out the questions below, and a store associate will be right out with your items.",
+  "deliveryHello" : "Hello",
+  "deliveryReadyHeader":"Items Ready for Delivery",
+  "deliveryReadyBody": "Item(s) in your order are ready to be delivery. A member of our team will contact you to schedule the delivery",
```

Comment:
Hi @sneha-nerekar,
There is still an issue with the message; "delivery" in the first sentence should be "delivered".
FROM: "Item(s) in your order are ready to be delivery. A member of our team will contact you to schedule the delivery"
TO: "Item(s) in your order are ready to be deliver**ed**. A member of our team will contact you to schedule the delivery"